source/content/debug-slow-performance.md

```diff
@@ -79,7 +79,7 @@ Cache misses are by nature slow - if something needs to be cached it's performed
 There are a large number of caches involved in every single request, including:
 
 - [Pantheon Global CDN](/global-cdn-caching) - Spread out across multiple servers, and the cache is not shared between servers.
-- [APC](/alternative-php-cache) - PHP has its own opcode cache, which is not shared between application containers.
+- [APCU](/apcu) - PHP has its own opcode cache, which is not shared between application containers.
```

Comment:
APCu isn't man opcode cache, though.

Reply:
Updated in https://github.com/pantheon-systems/documentation/pull/6044/commits/1d273f40730780fc32a1184b8e2705e3482fe298, leaving open for confirmation.